x-pack/filebeat/input/gcs/config.go

```diff
@@ -0,0 +1,52 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build !aix
+// +build !aix
+
+package gcs
+
+import (
+	"time"
+)
+
+// MaxWorkers, Poll & PollInterval can be configured at a global level,
+// which applies to all buckets , as well as at the container level.
+// Container level configurations will always override global level values.
```

Comment:
Add a description of the type before explaining the fields' behaviours.

```suggestion
// MaxWorkers, Poll & PollInterval can be configured at a global level,
// which applies to all buckets, as well as at the container level.
// Container level configurations will always override global level values.
```